### PR TITLE
Fix modeling docs case

### PR DESCRIPTION
**Pull request by borft**
Should fix Windows builds

### CODE REVIEW

File: docs/src/components/sphinx-page/index.tsx [hunk 1]
```diff
@@ -133,31 +133,51 @@ const SphinxPage = (props) => {
...
-                            a.href = a.href.replace(parts[2], '');
-                        }
-
-                        console.log(parts);
+                        // only remove the .html if local links, leave external links alone
+                        a.href = a.href.replace(/\.html/g, '');
 
+                        // remove the base URL, eg http://localhost:3000/ or https://staging.objectiv.io/docs/
+                        // from the href and then split it by '/'
+                        const relative_url = a.href.replace(currentSite[1].slice(0,-1) + baseUrl.baseUrl, '');
```
Comment (borft) [inline docs/src/components/sphinx-page/index.tsx:150]: this is where the magic happens

File: docs/src/components/sphinx-page/index.tsx [hunk 2]
```diff
@@ -133,31 +133,57 @@ const SphinxPage = (props) => {
...
+                            }
 
+                            // case 2
+                            if ( relative_parts[3].includes(relative_parts[2]) &&
+                                relative_parts[2][0].match(/[A-Z]/) ) {
+                                a.href = a.href.replace(relative_parts[1] + '/', '');
+                            }
+                        }
 
+                        // make sure to add a trailing slash to links with anchors that don't have it.
```
Comment (borft) [inline docs/src/components/sphinx-page/index.tsx:182]: @ivarpruijn this fixes the TransIP redirect issue

Comment (ivarpruijn) [reply docs/src/components/sphinx-page/index.tsx:182]: great!

**Comment by ivarpruijn** [comment]
LGTM!

**Pull request merged**